ingestion/src/metadata/ingestion/source/database/salesforce/metadata.py

```diff
@@ -231,6 +231,36 @@ def get_table_description(
             )
         return table_description if table_description else object_label
 
+    def get_column_description(
+        self, table_name: str, column_name: str
+    ) -> Optional[str]:
+        """
+        Method to get the column (field) description for Salesforce with the Tooling API.
+        """
+        column_description = None
+        try:
+            query = (
+                f"SELECT+Description+FROM+FieldDefinition+WHERE"
+                f"+EntityDefinition.QualifiedApiName='{table_name}'"
+                f"+AND+QualifiedApiName='{column_name}'"
+            )
+            result = self.client.toolingexecute(f"query/?q={query}")
```

Comment:
do we have to make this query for each column, can we just not get column description of all the columns within a table and the use the same?